features/step_definitions/login_steps.rb

```diff
@@ -0,0 +1,17 @@
+Given("following user exist") do |table|
+  table.hashes.each do |user|
+   FactoryBot.create(:user, user)
+  end
+end
+
+Given("I click {string}") do |button|
+  click_on button
+end
+
+When("I fill in {string} with {string}") do |input, value|
+  fill_in input, with: value
+end
+
+When("I click {string} button") do |button|
+  click_on(button)
```

Comment:
You have three steps that do the same, click a button, in you feature file word them all in the same way and then you only need 1 step for them all.